### ISSUE TITLE
Bug: CLI doesn't work

### ISSUE THREAD

**Issue by sfermigier** [litestar-org/litestar-fullstack#77]
### Description

`app` command doesn't work:

```
ModuleNotFoundError: No module named 'app.cli'; 'app' is not a package
```

### MCVE

```python
poetry install
app
```

### Litestar Version

2.0.0 9e67d91 (from `poetry.lock`)

### Platform

- [ ] Linux
- [X] Mac
- [ ] Windows
- [ ] Other (Please specify in the description above)

**Comment by cofin** [comment]
Just as a note, if you want to change the name from `app` to something else, you can edit [this line](https://github.com/cofin/litestar-fullstack/blob/main/pyproject.toml#L35) and change `app` to whatever you want your command name to be.  Afterwards, run `poetry install` to ensure the link is created in the venv.

**Comment by cofin** [comment]
I think we should make the documentation a bit more clear that you should run `make install`, this will ensure there is a clean environment at the start.

**Comment by swapbee** [comment]
Had to run `poetry shell` before running `app` then it works. 
Haven't tried after change to pdm.

**Comment by cofin** [comment]
Resolved by #92
